requirements.txt

```diff
@@ -5,7 +5,7 @@ numpy==1.20.0; python_version<"3.10.0"  # oldest supported by dwave-system
 numpy==1.22.0; python_version~="3.10.0"  # needed for scipy on windows
 oldest-supported-numpy; python_version>="3.11.0"
 
-scikit-learn==1.2.0; python_version >= '3.8'
-scikit-learn==1.0.2; python_version < '3.8'
+scikit-learn==1.2.1
+scipy>=1.7.3
```

Comment:
Rule of thumb, we want the versions in `requirements.txt` to be fixed for reproducibility, but as wide as possible in `setup.cfg` for flexibility.